myosuite/envs/myo/myobase/__init__.py

```diff
@@ -520,3 +522,40 @@ def register_env_with_variants(id, entry_point, max_episode_steps, kwargs):
                 'frame_skip': 5,
             }
     )
+
```

Comment:
Remove everything below this line. These aren't part of MyoSuite.

Baoding envs are in myochallenge folder